pkg/apis/serving/v1alpha1/route_lifecycle.go

```diff
@@ -126,7 +126,7 @@ func (rs *RouteStatus) MarkCertificateReady(name string) {
 	routeCondSet.Manage(rs).SetCondition(apis.Condition{
 		Type:     RouteConditionCertificateProvisioned,
 		Status:   corev1.ConditionTrue,
-		Severity: apis.ConditionSeverityWarning,
+		Severity: apis.ConditionSeverityInfo,
```

Comment:
I don't think we should change this Severity to `ConditionSerityInfo` because [`ConditionSeverityInfo`](https://github.com/knative/serving/blob/50cdf8762deae58ed89763adef26a88e7efecac7/vendor/knative.dev/pkg/apis/duck/v1alpha1/conditions_types.go#L57) means a failure of a condition type should be viewed as purely informational, and that things could still work. In this case, for the condition type `RouteConditionCertificateProvisioned`, it is not informational (even when it is ConditionTrue).

Basically I think `Severity` should tie to `ConditionType`, not `ConditionStatus`. We have an [example](https://github.com/knative/serving/blob/c6f7bc48faa2c882539d56719d5b23fdd653ea5c/pkg/reconciler/clusteringress/clusteringress_test.go#L214) which we set ConditionTrue for ConditionSeverityError.